### PR TITLE
Enable merge of hiera data and fix order of file change.

### PR DESCRIPTION
**Pull request by pmoranga**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -60,6 +63,16 @@
...
+      if $region {
+        file_line { 'region-on-awslogs':
+          path    => '/etc/awslogs/awscli.conf',
+          line    => "region = ${region}",
+          match   => '^region\s*=',
+          notify  => Service['awslogs'],
+          require => Package['awslogs'],
+        }
+      }
+
```
Comment (kemra102) [inline manifests/init.pp]: I don't think this module is the best place to do this kind of thing. Following the Unix philosophy of doing one thing and doing it well things like this should either be done in an awscli module or in your own `profiles` module. Alternatively this should probably be wrapped in a conditional that defaults to false to you have to run it off (maybe you aren't configuring/installing awscli on your box).

**Comment by kemra102** [comment]
Other than the above note this looks great.

File: manifests/init.pp (hunk 1 above)
Comment (pmoranga) [inline manifests/init.pp]: Hi kemra, I understand your point. But the file in the matter comes from the Amazon package "awslogs" and it defaults to  "us-east-1", so when I deploy a Amazon Linux server on a different zone, I would expect to be able to use the "region" parameter to set that my logs goes into that region. So I was just making the functionality complete of the region already defined parameter. I hope I could clarify my reasons.

Comment (kemra102) [inline manifests/init.pp]: That sounds reasonable to me.

**Comment by kemra102** [comment]
Merging, will look at releasing this to the Forge some time today.

**Pull request merged**